v2.6/usage/configuration/mtu.md

```diff
@@ -40,11 +40,29 @@ configured with VXLAN.
 ### Setting MTU for workload network interfaces
 
 It is the job of the network plugin to create new interfaces, the current
-major plugins are CNI and libnetwork.  Currently Docker and the Mesos Docker
-Containerizer integration use libnetwork which does **not** support setting MTU.
-CNI which is used by Kubernetes and the Mesos Unified Containerizer support
```

Comment:
I am not sure that we should lose this information here, which mentions that libnetwork does _not_ support setting MTU....unless that's incorrect. cc @ozdanborne @caseydavenport 

Also, the verb below should be plural... "support" --> "supports" (this verb is referring to CNI, which is singular).

Reply:
Oh oops, I'm just noticing now that indeed, we have a change in functionality and now libnetwork does support setting MTU, thanks to you. Great. So then, you can disregard my comment above.